grzyClothTool/Models/Drawable/GDrawable.cs

```diff
@@ -60,13 +60,38 @@ public string TypeName
         }
     }
 
+    private string _sexName;
+    public string SexName
+    {
+        get
+        {
+            _sexName ??= EnumHelper.GetSex(Sex ? 1 : 0);
+            return _sexName;
+        }
+        set
+        {
+            _sexName = value;
+        }
+    }
+
     [JsonIgnore]
     public List<string> AvailableTypes => IsProp ? EnumHelper.GetPropTypeList() : EnumHelper.GetDrawableTypeList();
+    public static List<string> AvailableSex => EnumHelper.GetSexTypeList();
 
     /// <returns>
     /// true(1) = male ped, false(0) = female ped
     /// </returns>
-    public bool Sex { get; set; }
+    private bool _sex;
```

Comment:
if there is enum introduced for sex, then I think it should be changed to use that enum everywhere instead of bool? This might be a lot of changes tho